packages/common/src/intl/intlHelpers.ts

```diff
@@ -18,6 +18,20 @@ export const useTranslation = (): ((
     id ? intl.formatMessage({ id: id as string }, values) : '';
 };
 
+export const useTranslationWithFallback = (): ((

```

Comment:
possibly useful elsewhere? In this case, just so that the breadcrumb doesn't look daft